spydrnet_physical/util/ConnectionPattern.py

```diff
@@ -159,7 +159,8 @@ def reset(self):
         self._connect = ConnectPointList(sizex=self.sizex,
                                          sizey=self.sizey)
 
-    def get_fishbone(self, width=None, height=None, steps=1, x_margin=(0, 0), y_margin=(0, 0)):
+    @staticmethod
+    def get_fishbone(width=None, height=None, steps=1, xbias=0, x_margin=(0, 0), y_margin=(0, 0)):
```

Comment:
remove None from the `width` and `height` arguments those are mandatory arguments